src/code-gen/markup-example.js

```diff
@@ -22,7 +22,8 @@ let mkExampleMarkup = (markup, model) => `
         var player = document.querySelector('pie-player');
 
         player.addEventListener('pie-player-ready', function(event){
-          player.controller = new pie.Controller(model.pies, pie.controllerMap);
+          var scoringProcessor = new pie.ScoringProcessorFactory().getProcessor(model);           
```

Comment:
Sure, no problem. We can pass the whole model to the Controller and pass it on to the factory.